maxim/logger/smolagents/client.py

```
@@ -0,0 +1,539 @@
+
+import contextvars
+import functools
+import inspect
+import logging
+import traceback
+import uuid
+from time import time
+from typing import Any, Callable, Optional, Union
+
+from ...logger import (
+    Generation,
```

Ruff (0.12.2)

`...logger.Generation` imported but unused

Remove unused import

(F401)